804-repositories.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Registries
```

Comment:
Is registries on a `weight` loss program?

Reply:
the file `804-` is redundant, as the content is now in file `200-` so I've purposely left it out of the website indexing here.